src/CommunityToolkit.Maui.Camera/CameraManager.shared.cs

```diff
@@ -52,6 +54,44 @@ public async Task<bool> ArePermissionsGranted()
 	/// <returns>A <see cref="ValueTask"/> that can be awaited.</returns>
 	public Task StartCameraPreview(CancellationToken token) => PlatformStartCameraPreview(token);
 
+
+	/// <summary>
+	/// Starts the video recording.
+	/// </summary>
+	/// <param name="token"><see cref="CancellationToken"/></param>
+	/// <returns>A <see cref="Task"/> that can be awaited.</returns>
+	/// <exception cref="CameraException"></exception>
+	public async Task<Stream> StartVideoRecording(CancellationToken token)
+	{
+		var cameraRequest = await Permissions.RequestAsync<Permissions.Camera>();
```

Comment:
Do we want to request permissions on behalf of the dev? 

I think it'd be better for the library, less to maintain, and more efficient to defer to the developer to request Camera + Microphone permissions. 

We would then update our documentation + samples to demonstrate how devs would request permission before starting the video recording.